rules/dead-code/src/NodeFinder/NextVariableUsageNodeFinder.php

```diff
@@ -80,4 +81,25 @@ public function find(Assign $assign): ?Node
 
         return $nextUsageOfVariable;
     }
+
+    private function findInParentExpression(Assign $assign, Variable $variable): bool
```

Comment:
`find*` usually return object/array, e.g. `findProduct` returns `Product`

Here should be `is*/has*` prefix... 

I hear you, this should be already in PHPStan rules :thinking: 

<br>

A better position for this method might be `BetterNodeFinder`

Reply:
How often do you hear voices? :D